resources/lang/id/model_validation/fulfillments.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/**
+ *    Copyright 2015-2017 ppy Pty. Ltd.
+ *
+ *    This file is part of osu!web. osu!web is distributed with the hope of
+ *    attracting more community contributions to the core ecosystem of osu!.
+ *
+ *    osu!web is free software: you can redistribute it and/or modify
+ *    it under the terms of the Affero GNU General Public License version 3
+ *    as published by the Free Software Foundation.
+ *
+ *    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+ *    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *    See the GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+return [
+    'username_change' => [
+        'only_one' => 'perubahan nama pengguna yang diperbolehkan hanya 1 kali per pesanan.',
+        'insufficient_paid' => 'Biaya penggantian nama pengguna melebihi jumlah yang telah dibayar (:expected > :actual)',
+        'reverting_username_mismatch' => 'Nama pengguna saat ini (:current) tidak sama dengan perubahan untuk dicabut (:username)',
+    ],
+    'supporter_tag' => [
+        'insufficient_paid' => 'Donasi kurang dari yang dibutuhkan untuk menghadiahkan supporter tag (:actual > :expected)',
```

Comment:
tag supporter.
terserah mau pakai yg mana, yang penting konsisten.
supporter tag / tag 'supporter'

Reply:
Will keep supporter tag. Thanks for the consistency reminder.